widgets/accordion/accordion.php

```diff
@@ -37,8 +37,30 @@ function initialize() {
 				)
 			)
 		);
+
+		add_action( 'siteorigin_widgets_enqueue_frontend_scripts_sow-accordion', array( $this, 'enqueue_widget_scripts' ) );
 	}
-	
+
+	function get_settings_form() {
+		return array(
+			'scrollto_after_change' => array(
+				'type'        => 'checkbox',
+				'label'       => __( 'Scroll user to top of the panel when opening panel', 'so-widgets-bundle' ),
```

Comment:
What do you think about renaming this label to:
"Scroll to top of the panel when opening panel"